mlflow/sklearn/__init__.py

```diff
@@ -365,7 +378,7 @@ def log_model(
         # log model
         mlflow.sklearn.log_model(sk_model, "sk_models")
     """
-    return Model.log(
+    Model.log(
```

Comment:
It seems [`Model.log()`](https://github.com/mlflow/mlflow/blob/d6ae84164987e2881cdffdce07f7dcf0e3d94aaa/mlflow/models/model.py#L140-L201) doesn't return any value. Maybe we can remove `return`.